net/acme-common/files/acme.init

```diff
@@ -140,26 +142,27 @@ load_globals() {
 
 start_service() {
 	mkdir -p $run_dir
-	exec 200>$run_dir/lock
-	if ! flock -n 200; then
-		log err "Another ACME instance is already running."
-		exit 1
-	fi
```

Comment:
I don't think we can assume that there will never be concurrent runs; a user could still execute `/etc/init.d/acme renew` - so let's keep this in the renew() function

Reply:
From my point of view, this is not necessary.
I have done a test during development of this change.
If I insert a `sleep 10` into the new `renew` function and call `renew` on two consoles with `/etc/init.d/acme renew`, then only one call can enter the renew function of the init script, the other on waits till the first one finished. This is because the procd already has a lock handling.

This applies to all functions that procd provides in an init script. Only one call of this init script is allowed. The others call to that init script wait until the other has finished the task.

Reply:
That sounds resonable. Will do that